spec/rails_helper.rb

```diff
@@ -72,9 +72,10 @@ class Analytics
       next if defined?($ran_asset_build)
       $ran_asset_build = true
       # rubocop:enable Style/GlobalVars
-      puts 'Bundling JavaScript and stylesheets...'
-      system 'WEBPACK_PORT= yarn build'
-      system 'yarn build:css'
+      print '                       Bundling JavaScript and stylesheets... '
+      system 'WEBPACK_PORT= yarn build > /dev/null 2>&1'
+      system 'yarn build:css > /dev/null 2>&1'
+      puts '✨ Done!'
```

Comment:
Are these changes intentional? What is all the extra spaces in the first line doing?

Reply:
Heh, I got tired of seeing the output of the yard build cluttering the rspec output. The spaces just push it out of the way so the visual scanning of the indent level of the output isn't interrupted. Ends up looking like this:
```
Analytics Regression
  FSMv2
    Happy path
                       Bundling JavaScript and stylesheets... ✨ Done!
      records all of the events
```
Happy to revert if you prefer!